tcms/testcases/models.py

```diff
@@ -55,6 +56,17 @@ class TestCase(models.Model, UrlMixin):
     requirement = models.CharField(max_length=255, blank=True, null=True)
     notes = models.TextField(blank=True, null=True)
     text = models.TextField(blank=True)
+    setup_duration = models.DurationField(db_index=True, null=True, blank=True)
+    testing_duration = models.DurationField(db_index=True, null=True, blank=True)
+
+    @property
+    def expected_duration(self):
+        if not self.setup_duration and not self.testing_duration:
+            return None
+        result = timedelta(0)
+        result += self.setup_duration or timedelta(0)
```

Comment:
This line does exactly the same thing as the if condition above. Remove the condition above.